code/src/pages/MoviePage.js

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from "react";
+
+import { Link, useParams } from "react-router-dom"; 
```

Comment:
this line can be moved up to be under the React import since they are both for things from nodes.

Reply:
Fixed!